packages/web-app-files/src/components/AppBar/ViewOptions.vue

```diff
@@ -37,20 +37,23 @@
       <oc-list>
         <li class="files-view-options-list-item oc-mb-m">
           <oc-switch
-            v-model="hiddenFilesShownModel"
+            v-model:checked="hiddenFilesShownModel"
             data-testid="files-switch-hidden-files"
             :label="$gettext('Show hidden files')"
+            @update:checked="updateHiddenFilesShownModel"
```

Comment:
To my understand of https://v3-migration.vuejs.org/breaking-changes/v-model.html#migration-strategy the `@update:checked` event listener is not needed - at least with your implementation of the handler. It just updates the `hiddenFilesShownModel` data prop to the event value. According to the migration guide that is already the case for `v-model:checked` ( / `v-model:whatever-custom-key`).